conf/base.config

```diff
@@ -44,6 +44,8 @@ process {
     cpus = { check_max( 10 * task.attempt, 'cpus' ) }
     memory = { check_max( 80.GB * task.attempt, 'memory' ) }
     time = { check_max( 8.h * task.attempt, 'time' ) }
+    errorStrategy = { task.exitStatus in [1,143,137,104] ? 'retry' : 'terminate' }
```

Comment:
The default for all processes is set on line 20:
```
errorStrategy = { task.exitStatus in [143,137] ? 'retry' : 'terminate' }
```

Instead of adding these lines to these specific processes, we can just update that line with the extra exit statuses and it will apply to everything.